goal_src/jak1/engine/camera/cam-states.gc

```diff
@@ -636,15 +636,15 @@
                       )
                     )
                   )
-              (set! (-> s5-0 y) (- (-> s5-0 y) (* (- f1-0) (-> *CAM_BILLY-bank* rot-speed))))
+              (set! (-> s5-0 y) (- (-> s5-0 y) (* (- f1-0) (* DISPLAY_FPS_RATIO (-> *CAM_BILLY-bank* rot-speed))))) ;; changed for high fps
               )
             )
           (cond
-            ((< (-> *CAM_BILLY-bank* rot-speed) (-> s5-0 y))
-             (set! (-> s5-0 y) (-> *CAM_BILLY-bank* rot-speed))
+            ((< (* DISPLAY_FPS_RATIO (-> *CAM_BILLY-bank* rot-speed)) (-> s5-0 y)) ;; changed for high fps
+             (set! (-> s5-0 y) (* DISPLAY_FPS_RATIO (-> *CAM_BILLY-bank* rot-speed))) ;; changed for high fps
              )
-            ((< (-> s5-0 y) (- (-> *CAM_BILLY-bank* rot-speed)))
-             (set! (-> s5-0 y) (- (-> *CAM_BILLY-bank* rot-speed)))
+            ((< (-> s5-0 y) (- (* DISPLAY_FPS_RATIO (-> *CAM_BILLY-bank* rot-speed)))) ;; changed for high fps
+             (set! (-> s5-0 y) (- (* DISPLAY_FPS_RATIO (-> *CAM_BILLY-bank* rot-speed)))) ;; changed for high fps
```

Comment:
I think this should actually take lag into account too. The fp camera has the same issue.

Reply:
Should be good now, seems to work correctly when lagging and the code is consistent with the other high FPS changes to cam-states.